presto-iceberg/src/test/java/com/facebook/presto/iceberg/IcebergDistributedTestBase.java

```diff
@@ -1666,6 +1666,24 @@ public void testDecimal(boolean decimalVectorReaderEnabled)
         }
     }
 
+    @Test
+    public void testRefsTable()
+    {
+        assertUpdate("CREATE TABLE test_table_references (id BIGINT)");
+        assertUpdate("INSERT INTO test_table_references VALUES (0), (1), (2)", 3);
+
+        Table icebergTable = loadTable("test_table_references");
+        icebergTable.manageSnapshots().createBranch("testBranch").commit();
+
+        assertUpdate("INSERT INTO test_table_references VALUES (0), (1), (2)", 3);
+
+        assertEquals(icebergTable.refs().size(), 2);
+        icebergTable.manageSnapshots().createTag("testTag", icebergTable.currentSnapshot().snapshotId()).commit();
+
+        assertEquals(icebergTable.refs().size(), 3);
+        assertQuery("SELECT count(*) FROM \"test_table_references$refs\"", "VALUES 3");
```

Comment:
Maybe we can add some more detailed assertions like follows:

```
assertQuery("SELECT * from \"test_table_references$refs\" where name = 'main' and type = 'BRANCH'",
                format("VALUES('%s', '%s', %s, %s, %s, %s)",
                        "main",
                        "BRANCH",
                        icebergTable.refs().get("main").snapshotId(),
                        icebergTable.refs().get("main").maxRefAgeMs(),
                        icebergTable.refs().get("main").minSnapshotsToKeep(),
                        icebergTable.refs().get("main").maxSnapshotAgeMs()));

        assertQuery("SELECT * from \"test_table_references$refs\" where type = 'TAG'",
                format("VALUES('%s', '%s', %s, %s, %s, %s)",
                        "testTag",
                        "TAG",
                        icebergTable.refs().get("testTag").snapshotId(),
                        icebergTable.refs().get("testTag").maxRefAgeMs(),
                        icebergTable.refs().get("testTag").minSnapshotsToKeep(),
                        icebergTable.refs().get("testTag").maxSnapshotAgeMs()));
```

Reply:
Sure, I have added detailed assertions for testTag & testBranch instead of the main branch. Please LMK if that's fine